From e8d209e9d66b13ad6eef5e84f7620781b0bbb6d8 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Ond=C5=99ej=20Nosek?= <onosek@redhat.com>
Date: Fri, 29 Aug 2025 01:07:49 +0200
Subject: [PATCH 13/13] `srpm`: man page generation fixed
MIME-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit

Usage wasn't generated properly for this command. The usage was
overridden with fixed text instead of automatic generation.

Relates: #751

Signed-off-by: Ondřej Nosek <onosek@redhat.com>
---
 pyrpkg/cli.py | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/pyrpkg/cli.py b/pyrpkg/cli.py
index 10c6fd4..9fbb166 100644
--- a/pyrpkg/cli.py
+++ b/pyrpkg/cli.py
@@ -1548,9 +1548,10 @@ class cliClient(object):
         srpm_parser = self.subparsers.add_parser(
             'srpm', help='Create a source rpm',
             parents=[self.rpm_parser_common],
-            usage='Create a source rpm',
             formatter_class=argparse.RawDescriptionHelpFormatter,
             description=textwrap.dedent("""
+                Create a source rpm
+
                 This command wraps "rpmbuild -bs", roughly equivalent to:
 
                   rpmbuild -bs mypackage.spec \\
-- 
2.51.0

